hw/drivers/sensors/lis2dw12/src/lis2dw12.c

```diff
@@ -1717,6 +1805,18 @@ lis2dw12_sensor_set_notification(struct sensor *sensor, sensor_event_type_t type
         return rc;
     }
 
+    /* enable double tap detection in wake_up_ths */
+    if(type == SENSOR_EVENT_TYPE_DOUBLE_TAP) {
+        wake_up_ths = lis2dw12->cfg.wake_up_ths;
+        wake_up_ths |= LIS2DW12_WAKE_THS_SINGLE_DOUBLE_TAP;
+
+        rc = lis2dw12_set_wake_up_ths(itf, wake_up_ths);
```

Comment:
Lets mask using LIS2DW12_WAKE_THS_THS, or mask in the function call itself?

Reply:
Masking in the function call is probably the right way to do it.